src/main/resources/application.yaml

```diff
@@ -189,7 +189,7 @@ SERVICE_AUTH_MICROSERVICE: sptribs_case_api
 
 toggle:
   enable_entitlement_email: ${ENABLE_ENTITLEMENT_EMAIL:true}
-  enable_jrd_api_v2: ${ENABLE_JRD_API_V2:false}
+  enable_jrd_api_v2: ${ENABLE_JRD_API_V2:true}
```

Comment:
Enabled to allow judges list to be populated on preview. Will remove once testing is complete.